qmd/_dictionary-version.qmd

```diff
@@ -6,4 +6,4 @@ dictionary_version_pattern <- "ODM_dictionary_(.*?).xlsx"
 
 dictionary_version_used <- regmatches(dictionary_full_file_name, regexec(dictionary_version_pattern, dictionary_full_file_name))[[1]][2]
 ```
-ODM version `r dictionary_version_used`.
\ No newline at end of file
+ODM `r dictionary_version_used`.
```

Comment:
In the quarto.yml you use Dictionary to specify the dictionary version. I think ODM should be changed to Dictionary nary to be standard across pages.

Reply:
Agree. Can you make that change?